.travis.yml

```diff
@@ -17,7 +17,7 @@ matrix:
     env: PYV=3.8
   - python: pypy3
     before_install:
-      apt-get install libssl1.1
+    - sudo apt-get install libssl1.1
```

Comment:
I'd try sudo apt-get update && sudo apt-get install libssl1.1 - often the cache is empty in systems like this.

Reply:
Sadly the problem is there's no libssl1.1 for xenial: the earliest supported distribution that has it in the standard repository is ubuntu 18.
Ubuntu 16 is nearing its end of support.